sql/item_latents.sql

```diff
@@ -35,7 +35,11 @@ INSERT INTO `item_latents` VALUES (10679,11,10,10,0);
 INSERT INTO `item_latents` VALUES (10719,12,10,11,0);
 INSERT INTO `item_latents` VALUES (10719,13,10,11,0);
 INSERT INTO `item_latents` VALUES (10719,14,10,11,0);
-INSERT INTO `item_latents` VALUES (10730,63,10,13,56);  -- WAR AF2 +2 Feet Reduces defense penalty by 10% if Berserk Active
+
+-- -------------------------------------------------------
```

Comment:
Just looking at the rest, do we really need these two long lines of comments to create a header here? I'd say let's keep it simple to a single-line comment (and if the rest of the things in latents do that, nuke it from orbit.

Reply:
They do. let this pass, and later tomorrow I'll kill the dashed lines above and below the item name.

This is a case of past conversations everyone forgot about. In the dark days before tooling.

Reply:
as promised: #4505